src/runtime_src/core/tools/xbmgmt2/flash/flasher.cpp

```diff
@@ -187,6 +190,23 @@ int Flasher::upgradeFirmware(const std::string& flasherType,
         }
         break;
     }
+    case OSPI_XGQ:
+    {
+        XGQ_Flasher xgq_flasher(m_device);
+        if (primary == nullptr)
+        {
+            std::cout << "ERROR: OSPI XGQ mode does not support reverting to MFG." << std::endl;
+        }
```

Comment:
Ditto convention and at least keep opening curly brace at some line as conditional

Reply:
removed {}